### DISCUSSION TITLE
Self Learning Tracking problem

### DISCUSSION THREAD

**Discussion by iamshivampakade** [github/india#339] [Students]
I have completed my first module in self learning sessions but it's showing progress instead of complete at FRT Portal

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together!
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by divyavemasani2910** [comment]
> I have completed my first module in self learning sessions but it's showing progress instead of complete at FRT Portal

I am facing the same issue

**Reply by iamshivampakade** [reply]
I think they need some time to track our module's progress

**Reply by ghost** [reply]
But without knowing whether the module is completed or not we can't go further. Later it might be a problem.

**Reply by PRIYA-S-102** [reply]
have you completed your FRT Internship project if yes help me for completing.
mail me at princessrajpriya26@gmail.com
